app/dependencies.php

```diff
@@ -339,12 +343,19 @@ static function (ContainerInterface $c): Matching\Adapter {
 
             $sendersLocator = new SendersLocator(
                 [
+                    // Outbound for ClaimBot; Redis queue in Production.
                     Messages\Donation::class => [ClaimBotTransport::class],
+
+                    // Outbound, priority, for MatchBot worker; SQS queue in Production.
+                    // `CharityUpdated` does call out to Salesforce, to read data, but it's rarer and
+                    // occasionally more time-sensitive than the group below which push data.
                     CharityUpdated::class => [TransportInterface::class],
                     StripePayout::class => [TransportInterface::class],
```

Comment:
this is sort of confusing since both queues implement TransportInterface so it isn't immediatly obvious which one this refers to. Would be clearer IMHO if we have two new interfaces to mark the two queues. Or don't use classlike names as the keys in the senders locator and just use plain strings. As suggested on the infra PR could be clearer if they're named something like "high-priority-transport" and "low-priority-transport".

Reply:
I'm not sure I completely understand what that would look like, and how `SendersLocator` would get a usable transport. Would we start using the plain strings as keys on the main DI container?

Reply:
Yes, there's no rule that says the lookup keys in the DI locator have to be types or class-strings. If you look at mentions of e.g. SalesforcePushTransport it's only used in dependancies.php and in contexts where all we need is a unique string - so could just be a constant:

![image](https://github.com/user-attachments/assets/6d1cc108-1061-4d48-a446-0750b217a577)

If the interface was used on the right of line 514 I think that would crash with a TypeError it doesn't actually construct an instance of the interface.